xpk.py

```diff
@@ -3834,6 +3834,20 @@ def workload_create(args) -> int:
 
   xpk_print("Starting workload create", flush=True)
 
+  metadata_configmap_name = f'{args.cluster}-{_CLUSTER_METADATA_CONFIGMAP}'
+  cluster_config_map = get_cluster_configmap(args, metadata_configmap_name)
+  cluster_xpk_version = None
+  if cluster_config_map is None:
+    xpk_print(f"Warning: Unable to find ConfigMap: {metadata_configmap_name} for the cluster. "
+              "We recommend to upgrade your cluster by running `xpk cluster create`.")
+  else:
+    cluster_xpk_version = cluster_config_map.get("xpk_version")
+  if cluster_xpk_version is not None and cluster_xpk_version < xpk_current_version:
```

Comment:
what do you think if we change the check to `cluster_xpk_version != xpk_current_version`.

Probably if the cluster version > xpk_current_version, we can expect potentiall issues too?

Reply:
Yes, that sounds good. I will also update the warning message in that case.